### PR TITLE
chore: fix end margin for album/artist list

### LINKED ISSUE #205: Cutted off buttons on album/artist list
The buttons are not aligned properly, can be fixed by setting margin end here:

https://github.com/vixalien/muzika/blob/bb85dad038dad3a9b2e8d3c756eafd77c4eb3f65/src/components/playlist/columnview/columns/menu.ts#L56-L57

`box.set_margin_end(14);`

Before
Before

After
After

### PR DESCRIPTION
**Pull request by diegopvlk**
fix #205

### CODE REVIEW

File: src/components/playlist/columnview/columns/menu.ts [hunk 1]
```diff
@@ -54,6 +54,7 @@ class ExtraMenuButtons extends Adw.Bin {
     });
 
     const box = Gtk.Box.new(Gtk.Orientation.HORIZONTAL, 3);
```
Comment (vixalien) [inline src/components/playlist/columnview/columns/menu.ts:56]: You can use easier syntax 

```js
const box = new Gtk.Box({
  orientation: ...,
  spacing: ...,
  margin_end: ...
});
```

**Review by vixalien — CHANGES_REQUESTED** [review]

File: src/components/playlist/columnview/columns/menu.ts [hunk 2]
```diff
@@ -53,7 +53,11 @@ class ExtraMenuButtons extends Adw.Bin {
       this.like_button_toggled(false);
     });
 
-    const box = Gtk.Box.new(Gtk.Orientation.HORIZONTAL, 3);
+    const box = new Gtk.Box({
+      orientation: Gtk.Orientation.HORIZONTAL,
+      spacing: 3,
+      margin_end: 14
```
Comment (vixalien) [inline src/components/playlist/columnview/columns/menu.ts:59]: Please fix the lint error, by adding a trailing semicolon here

**Review by vixalien — CHANGES_REQUESTED** [review]

File: src/components/playlist/columnview/columns/menu.ts (hunk 2 above)
Comment (DaPigGuy) [reply src/components/playlist/columnview/columns/menu.ts:59]: Yes

**Review by DaPigGuy — APPROVED** [review]

**Review by vixalien — APPROVED** [review]

**Comment by vixalien** [comment]
Thank you!

**Pull request merged**